x-pack/plugins/observability_solution/synthetics/common/requests/get_certs_request_body.ts

```diff
@@ -147,6 +147,10 @@ export const getCertsRequestBody = ({
         'tls.server.hash.sha256',
         'tls.server.x509.not_after',
         'tls.server.x509.not_before',
+        'service',
+        'labels',
+        'tags',
```

Comment:
@shahzad31 A random question: What do we save in the "tags" field in the uptime/synthetics rule? (I mean the data that we save in the AAD.)

Reply:
in AAD we save monitor.tags which contains monitor tags, i think root tags are always rule tags. we aren't changing those.